features/waarden.feature

```diff
@@ -0,0 +1,194 @@
+
+  N.B. De provider van de API kan een maximum aantal op te nemen waarden bepalen.
+  N.B. Niet elke waarde bij het WOZ-object hoeft te komen uit een beschikking op naam van de actuele belanghebbende(n).
+
+  indicatieLopendGeding krijgt de waarde true wanneer StatusBeschikking een van de volgende waarden heeft:
+    - bezwaar ingediend
+    - beroep aangetekend
+    - hoger beroep aangetekend
+    - cassatie ingesteld
+
+
+  Scenario: meerdere jaren met zelfde eigenaar en zonder geding
+    Gegeven WOZ-object met objectnummer "051800823525" heeft de volgende beschikkingen voor eigenaar R. Geels   :
+      | Peildatum  | WOZ-waarde   | Belanghebbende | Ingangsdatum | Datum beschikking |
+      | 01-01-2019 | 431.000 euro | R. Geels       | 01-01-2020   | 11-02-2020        |
+      | 01-01-2018 | 390.000 euro | R. Geels       | 01-01-2019   | 12-02-2019        |
+      | 01-01-2017 | 340.000 euro | R. Geels       | 01-01-2018   | 13-02-2018        |
+      | 01-01-2016 | 340.000 euro | R. Geels       | 01-01-2017   | 14-02-2017        |
+      | 01-01-2015 | 310.000 euro | R. Geels       | 01-01-2016   | 16-02-2016        |
+      | 01-01-2014 | 308.000 euro | R. Geels       | 01-01-2015   | 17-02-2015        |
+    Als het WOZ object wordt opgevraagd met /wozobjecten/051800823525
+    Dan bevat het antwoord:
+    """
+    "waarden": [
+      {
+        "vastgesteldeWaarde": 431000,
+        "waardepeildatum": "2019-01-01"
+      },
+      {
+        "vastgesteldeWaarde": 390000,
+        "waardepeildatum": "2018-01-01"
+      },
+      {
+        "vastgesteldeWaarde": 340000,
+        "waardepeildatum": "2017-01-01"
+      },
+      {
+        "vastgesteldeWaarde": 340000,
+        "waardepeildatum": "2016-01-01"
+      },
+      {
+        "vastgesteldeWaarde": 310000,
+        "waardepeildatum": "2015-01-01"
+      },
+      {
+        "vastgesteldeWaarde": 308000,
+        "waardepeildatum": "2014-01-01"
+      }
+    ]
+    """
+
+  Scenario: WOZ-object wijzigt gedurende een jaar van eigenaar
+    Gegeven WOZ-object met objectnummer "002500003118" heeft de volgende beschikkingen voor P. Keizer:
+      | Peildatum  | WOZ-waarde   | Belanghebbende | Ingangsdatum | Datum beschikking |
+      | 01-01-2018 | 160.000 euro | P. Keizer      | 01-01-2019   | 12-02-2019        |
+      | 01-01-2017 | 155.000 euro | P. Keizer      | 01-01-2018   | 13-02-2018        |
+      | 01-01-2016 | 149.000 euro | P. Keizer      | 01-01-2017   | 14-02-2017        |
+    En dit WOZ-object heeft per 17-03-2019 een nieuwe eigenaar W. Rijnsbergen
+    En WOZ-object met objectnummer "002500003118" heeft de volgende beschikkingen voor W. Rijnsbergen
+      | Peildatum  | WOZ-waarde   | Belanghebbende | Ingangsdatum | Datum beschikking |
+      | 01-01-2019 | 171.000 euro | W. Rijnsbergen | 01-01-2020   | 11-02-2020        |
+      | 01-01-2018 | 160.000 euro | W. Rijnsbergen | 17-03-2019   | 06-04-2019        |
```

Comment:
Als hier 165.000 euro had gestaan, zou ik dan dit bedrag hebben gezien?

Reply:
bedrag wijzigt alleen wanneer de nieuwe eigenaar bezwaar heeft ingediend en is goedgekeurd